### PR TITLE
UI tweaks for AAF mode.

### PR DESCRIPTION
**Pull request by carlohamalainen**
* Setting to disable user registration.
* Setting to disable user account management.
* Use AAF Rapid Connect URL for login button.

And also:

* Instead of raising a ValueError on a replay attack, wipe the session info, log it, and redirect the user to the front page.

### CODE REVIEW

File: tardis/tardis_portal/views.py [hunk 1]
```diff
@@ -219,6 +219,8 @@ def index(request):
         .exclude(public_access=Experiment.PUBLIC_ACCESS_NONE)\
         .order_by('-update_time')[:limit]
     c['public_experiments'] = public_experiments
+    c['RAPID_CONNECT_ENABLED']   = settings.RAPID_CONNECT_ENABLED
```
Comment (grischa) [inline tardis/tardis_portal/views.py]: Codacy Issue found: [Exactly one space required before assignment](https://www.codacy.com/app/mytardis/mytardis/file/743149942/issues/source?bid=2125231&fileBranchId=2440569#l222)

Comment (grischa) [inline tardis/tardis_portal/views.py]: Codacy Issue found: [Exactly one space required before assignment](https://www.codacy.com/app/mytardis/mytardis/file/768770113/issues/source?bid=2125231&fileBranchId=2440569#l222)

**Comment by carlohamalainen** [comment]
Is this good to go?

File: tardis/tardis_portal/templates/tardis_portal/index.html [hunk 2]
```diff
@@ -13,9 +13,12 @@
     <h1>{{site_title|default:'MyTardis'}} Data Store</h1>
   </div>
   <p>
-    {% if not is_authenticated %}
```
Comment (grischa) [inline tardis/tardis_portal/templates/tardis_portal/index.html]: @carlohamalainen I shouldn't have pulled this as is, it looks like you removed the `is_authenticated` check. Can you have a look whether this change would work for you, too: https://github.com/mytardis/mytardis/pull/476 ?

**Pull request merged**